server/routes/payment.js

```diff
@@ -1,12 +1,7 @@
 const express = require("express");
 const router = express.Router();
 const knex = require("../utils/dbConfig");
-const Razorpay = require("razorpay");
-
-var razorpay = new Razorpay({
-  key_id: process.env.RAZOR_PAY_KEY,
-  key_secret: process.env.RAZOR_PAY_KEY_SECREAT,
-});
+const payment = require("../utils/paymentMode");
```

Comment:
can we name it as paymentGateway instead of paymentMode?

Reply:
fiixed